test/framework/src/main/java/org/elasticsearch/test/ESSingleNodeTestCase.java

```diff
@@ -355,4 +355,19 @@ protected boolean forbidPrivateIndexSettings() {
         return true;
     }
 
+
+    /**
+     * waits until all shard initialization is completed.
+     *
+     * @throws IOException
+     */
+    protected void ensureNoInitializingShards() throws IOException {
+        ClusterHealthResponse actionGet = client().admin()
+            .cluster()
+            .health(Requests.clusterHealthRequest("_all").timeout(TimeValue.timeValueSeconds(70)).waitForNoInitializingShards(true))
```

Comment:
Why `70`? Could you add a comment explaining it?

Reply:
good question, I took it over from ESRestTestCase, but there is no explanation either.

I removed it now, the default is `30s`, which should work. If not, it can be increased as needed.